corehq/apps/domain/templates/domain/admin/info_basic.html

```diff
@@ -2,8 +2,11 @@
 {% load crispy_forms_tags %}
 {% load hq_shared_tags %}
 
+
 {% requirejs_main 'domain/js/info_basic' %}
 
 {% block page_content %}
+  <link rel="stylesheet" href="{% static '@mapbox/mapbox-gl-geocoder/dist/mapbox-gl-geocoder.css' %}" type="text/css" />
```

Comment:
There's a stylesheets block you can put this in instead, so it gets included with other css [here](https://github.com/dimagi/commcare-hq/blob/92656804df7513dc95826572d7c270855fbf293f/corehq/apps/hqwebapp/templates/hqwebapp/base.html#L56).